src/drive/ducks/files/Toolbar.jsx

```diff
@@ -55,11 +58,93 @@ class Toolbar extends Component {
       canUpload,
       uploadFiles,
       downloadAll,
-      leaveFolder
+      leaveFolder,
+      breakpoints: { isMobile }
     } = this.props
     const notRootfolder = displayedFolder && displayedFolder.id !== ROOT_DIR_ID
     const hasWriteAccess =
       canUpload && (!shared.withMe || shared.sharingType === 'master-master')
+
```

Comment:
This looks like a lot of changes, but in reality this section of the component was already there — I'm just putting it in a variable so I can easily wrap it (or not) later.